apps/web/playwright/lib/testUtils.ts

```diff
@@ -358,6 +361,9 @@ export async function doOnOrgDomain(
     "x-cal-force-slug": orgSlug,
   });
   await callback({ page });
+  await page.setExtraHTTPHeaders({
+    "x-cal-force-slug": "",
+  });
```

Comment:
Reset x-cal-force-slug back so that code following the function call is unaffected by `x-cal-force-slug`